responses/00-overview.md

```diff
@@ -1,6 +1,6 @@
 **Welcome to the second installment of "introduction to data pipelines"** at USGS, @{{ user.username }}!! :sparkles:
 
-We're assuming you were able to navigate through the [intro-to-pipelines](https://lab.github.com/USGS-R/intro-to-pipelines) course and that you learned a few things about organizing your code for readability, re-use, and collaboration. You were also introduced to two key things through the `remake.yml`: a way to program connections between functions and files, and the concept of a dependency manager that skips parts of the workflow that don't need to be re-run. 
+We're assuming you were able to navigate through the [intro-to-pipelines](https://lab.github.com/USGS-R/intro-to-pipelines) course and that you learned a few things about organizing your code for readability, re-use, and collaboration. You were also introduced to two key things through the makefile: a way to program connections between functions and files, and the concept of a dependency manager that skips parts of the workflow that don't need to be re-run. 
```

Comment:
Good catch - I might make a second issue to update all of the links to the other courses because I don't yet know what they will be. Is it based on this line? https://github.com/USGS-R/ds-pipelines-1-course/blob/master/config.yml#L1

Reply:
Seems likely that it's that field that matters, yes, but I haven't found the docs to confirm or refute.

Reply:
captured in #5 for now and will check once I have the courses up for testing